config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+get 'menu/index'
+
+root 'menu#index'
```

Comment:
Please use the proper syntax when writing routes, and make sure that you use resourceful routing.